### DISCUSSION TITLE
Nextjs typescript error Cannot read property 'referencePaths' of undefined

### DISCUSSION THREAD

**Discussion by mi3lix9** [ben-rogerson/twin.examples#46] [Q&A] [answered]
I added twin to my project but I get this Error: 

`Cannot read property 'referencePaths' of undefined
    at transformFile.next (<anonymous>)
    at run.next (<anonymous>)`

**Comment by ben-rogerson** [comment]
Could you include a sample project or minimal reproduction steps?

**Reply by mi3lix9** [reply]
https://github.com/Mi3liX9/project-kushk/tree/beta

**Comment by mi3lix9** [comment]
When I remove the type from the component (:React.FC) the error disappear 🤔

**Reply by ben-rogerson** [reply]
I see some recent commits, did you manage to get this working?

**Reply by mi3lix9** [reply]
Yes, I jus moved the code of twin to the buttom of babel file

**Comment by mi3lix9** [comment, marked as answer]
In the babel file I added some TypeScript babel plugins so **I moved the config of Styled Components and Twin to the buttom and it worked!**
